service/src/main/java/org/folio/rest/workflow/controller/AdminController.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.rest.workflow.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/admin")
```

Comment:
I would prefer actuator health endpoint.

https://stackoverflow.com/questions/52654763/changing-spring-boot-actuator-health-end-point-to-a-custom-end-point#:~:text=The%20default%20actuator%20health,endpoint%20would%20be%20http%3A%2F%2Flocalhost%3A8080%2Factuator%2Fhealth.

https://www.baeldung.com/spring-boot-actuators